task_sdk/src/airflow/sdk/definitions/mappedoperator.py

```diff
@@ -63,13 +63,13 @@
     from airflow.models.abstractoperator import (
         TaskStateChangeCallback,
     )
-    from airflow.models.baseoperatorlink import BaseOperatorLink
     from airflow.models.expandinput import (
         ExpandInput,
         OperatorExpandArgument,
         OperatorExpandKwargsArgument,
     )
     from airflow.models.xcom_arg import XComArg
+    from airflow.sdk import BaseOperatorLink
```

Comment:
From inside the SDK we probably want to import from the definitions directly for consistency.

Reply:
Nice observation. Made the change